docs/usage/general.rst

```diff
@@ -5,6 +5,13 @@ Borg consists of a number of commands. Each command accepts
 a number of arguments and options and interprets various environment variables.
 The following sections will describe each command in detail.
 
+.. container:: experimental
+
+   Experimental features are marked with red-orange stripes on the sides, like this paragraph.
```

Comment:
well, we can do that, but I guess we'll also have `(experimental)` or so as textual marker?